conf/modules.config

```diff
@@ -1286,6 +1289,54 @@ process {
         ]
     }
 
+    withName: ".*MERGE_LIBRARIES_DAMAGE_MANIPULATION:SAMTOOLS_MERGE_LIBRARIES" {
+        tag = { "${meta.reference}|${meta.sample_id}|${meta.damage_manipulation}" }
+        ext.prefix = { "${meta.sample_id}_${meta.reference}_${meta.damage_manipulation}_unsorted" }
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: ".*MERGE_LIBRARIES_DAMAGE_MANIPULATION:SAMTOOLS_SORT_MERGED_LIBRARIES" {
+        tag = { "${meta.reference}|${meta.sample_id}|${meta.damage_manipulation}" }
+        ext.prefix = { "${meta.sample_id}_${meta.reference}_${meta.damage_manipulation}" }
```

Comment:
I think naming the outputs `${meta.sample_id}_${meta.damage_manipulation}_${meta.reference}` would be more intuitive, since the damage manipulation is not a reference attribute but a consequence of the sample attributes.